pkgs/top-level/all-packages.nix

```diff
@@ -36275,4 +36275,6 @@ with pkgs;
   swift-corelibs-libdispatch = callPackage ../development/libraries/swift-corelibs-libdispatch { };
 
   swaysettings = callPackage ../applications/misc/swaysettings { };
+
+  pista = callPackage ../misc/pista { };
```

Comment:
There is probably a better location to put this than at the very end of the file, but I am not really sure.

Reply:
It doesn't seem to be alphabetically sorted or grouped :thinking: